src/test/components/TooltipMarker.test.js

```diff
@@ -67,323 +67,313 @@ describe('TooltipMarker', function() {
 
     // Now add some markers to the profile.
     // Enumerate through all of the switch arms of the tooltip for coverage.
-    addMarkersToThreadWithCorrespondingSamples(
```

Comment:
this is mostly some indentation change here; I only changed that it doesn't take a second parameter anymore.